### DISCUSSION TITLE
Modelling and inventory communities' definitions of land greenhouse gas emissions

### DISCUSSION THREAD

**Discussion by irismuz** [ESCOMP/CTSM#1944] [Ideas]
There is a disparity between the land classification used in the modelling and inventory communities (kindly see the table below) which makes it challenging to compare the GHG emissions of different categories (anthropogenic and managed land categories on one hand and natural and unmanaged land categories on the other). UNFCCC classification is commonly used for countries' GHG emissions reporting.

![Land_CO2_emissions](https://user-images.githubusercontent.com/63979539/214382142-6dd9ac4e-cdb8-4ac3-bee3-7a0a4c10ea1c.png)

In the current CTSM configuration, forests are considered part of the natural land category, and there are no managed wetland (e.g. peat extraction areas and drained mires) and no unmanaged wetland categories. It thus seems that assigning bare soil emissions from CTSM to the unmanaged land category could be the option for separating managed and unmanaged land categories defined by the inventory community. The CTSM output provides NPP and GPP at the PFT level, however, NEE is indexed on the grid cell level.

@danicalombardozzi noted that NEE is not a PFT-level variable since it includes processes like soil respiration, fire fluxes and harvest fluxes that are calculated on the column level or coarser. Bare soil and natural vegetation share a soil column so the separation of the column-level fluxes from bare soil and natural vegetation is not straightforward. 

As mentioned by @ekluzek, this insight is both an infrastructure issue as well as a science one.

**Comment by korsbakken** [comment]
A bit of context might be useful to understand the issue. The example land categories (bottom row) in the table are how land categories are assigned to managed and unmanaged land in Norway. Other countries may differ, but the case of Norway can be instructive.

In Norway, all forests are categorized as "managed land" in the GHG inventory, the rationale being that nearly all forest in Norway is not "pristine", but heavily influenced by past and present human activities, including forestry, mixed forestry/grazing, and reforestation of former pasture and marginal croplands. Much non-forested, non-grass marginal land (mostly rocks and bare soil) on the other hand is categorized as "unmanaged". To differentiate between the fluxes from managed and unmanaged land in CTSM CO2 fluxes, it would therefore be useful to be able to distinguish between vegetated and bare soil for all relevant fluxes.

The challenge is that some fluxes (notably most soil-related fluxes) are parametrized by column, landunit or even just gridcell, and not by PFT. The former three do not distinguish between bare and vegetated soils, and therefore it is not possible to cleanly split total CO2 fluxes between bare soil and vegetated land either.

A second challenge is that it is difficult to isolate the CO2 fluxes associated with wetlands, which would also be needed to consistently distinguish between managed and unmanaged land. However, in this case, I'm not sure there is much that can be done, at least not for the specific case of Norway. The main issue there is that there are both managed and unmanaged wetlands, typically in patches that would require 10- to 100-meter resolution to capture, so I don't think it would be feasible to distinguish managed and unmanaged wetlands in CTSM. Fortunately it is a relatively marginal problem in the case of Norway. Forests vs. bare soil however is very much not a marginal issue.

**Reply by rosiealice** [reply]
Hi @irismuz & @korsbakken,

That is an interesting point r.e. the column level NEE undermining our attempts to get at the patch/pft level NEE fluxes. 

Remembering what has happened to managed forests (in terms of re growth trajectories) is one of the motivations for FATES, where we are currently designing the LULUC implementation (that will likely have multiple stages). It would be interesting to consider what our choices mean wrt the comparison with UNFCCC. I will try and find the relevant thread and link it here.
